x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/700_rrf_retriever_aggs.yml

```diff
@@ -0,0 +1,156 @@
+setup:
+  - requires:
+      cluster_features: 'rrf_retriever_supported'
```

Comment:
Since we shouldn't use versions anymore, should I introduce a new `cluster_feature` for this set of yml tests?

Reply:
@pmpailis this is a pretty big feature. Adding a cluster feature, not just a test feature, might be a good idea.

Reply:
Thanks Ben! Added in https://github.com/elastic/elasticsearch/pull/112648/commits/123f99602ca626a18737563e57498bae5ec34f62

Reply:
(I'm open to suggestions for the feature name :D )